get-started/index.md

```diff
@@ -336,8 +336,8 @@ ENV JAVA_OPTS="-javaagent:/deployments/app/cryostat-agent.jar"
 ```
 
 Next we must rebuild our container image. This is specific to your application but will likely look something like `docker build -t docker.io/myorg/myapp:latest -f src/main/docker/Dockerfile .`.
-Push that updated image or otherwise get it updated in your Kubernetes registry, then modify your application `Deployment` to supply JVM system properties or environment variables configuring
-the Cryostat Agent:
+**Push** that updated image or otherwise get it updated in your Kubernetes registry, then modify your application `Deployment` to supply **JVM** system properties or environment variables configuring
```

Comment:
Bolding "**Push**" like this makes it look like this is a particular term relating to Cryostat or OpenShift or something, but it isn't. I think this should just be unformatted text.